### PR TITLE
Switch the ghost and golem buildings

### PR DESCRIPTION
**Pull request by Fabrice-TIERCELIN**
Re @misiokles,

This PR switches the buildings of the Bone Golem and of the Ghosts. The Canyon is very:

- Huge
- Impressive
- Beautiful

It is more logical that it is related to a big creature. Also, the Temple of Death is tiny. I have also adjusted the name of the buildings.

As it is an important change, I incremented the version, but it remains compatible with previous version. Just, the further maps will be created slightly differently, but without error. I had added sounds on buildings in the previous PR to help the players to recognize the respective buildings.

I had to adapt the translations. @heroesiiifan, warn me if there are mistakes in the German translation. @by003, warn me if there are mistakes in the Chinese translation.

### CODE REVIEW

**Comment by Fabrice-TIERCELIN** [comment]
@ToRRent1812, @czapnaper, can you proofread my polish translation update, please?

https://github.com/vcmi-mods/death-valley-town/pull/8/files#diff-9ac671382451cfb7eac4dd409a5c781ad05ea4a01bd4e097804383a8cb9ad018

File: death-valley-town/Content/translation/deathvalley/polish.json [hunk 1]
```diff
@@ -17,28 +17,28 @@
...
+    "building.death-valley-town.deathvalley.dwelling7.name": "Kanion Śmierci",
     "building.death-valley-town.deathvalley.dwellingUpgrade1.description": "Ulepszony cmentarz pozwala rekrutować Grobowe bestie.",
     "building.death-valley-town.deathvalley.dwellingUpgrade1.name": "Ul. Cmentarz",
     "building.death-valley-town.deathvalley.dwellingUpgrade2.description": "Ulepszony kopiec grobowy pozwala rekrutować Łuczników grozy.",
     "building.death-valley-town.deathvalley.dwellingUpgrade2.name": "Ul. Kopiec grobowy",
     "building.death-valley-town.deathvalley.dwellingUpgrade3.description": "Ulepszona wieża kościelna pozwala rekrutować Przegniłych.",
     "building.death-valley-town.deathvalley.dwellingUpgrade3.name": "Ul. Wieża kościelna",
-    "building.death-valley-town.deathvalley.dwellingUpgrade4.description": "Ulepszony nawiedzony wąwóz pozwala rekrutować Banshee.",
-    "building.death-valley-town.deathvalley.dwellingUpgrade4.name": "Ul. Nawiedzony wąwóz",
+    "building.death-valley-town.deathvalley.dwellingUpgrade4.description": "Ulepszony Nawiedzona Świątynia pozwala rekrutować Banshee.",
```
Comment (ToRRent1812) [inline death-valley-town/Content/translation/deathvalley/polish.json:34]: Ulepszony -> Ulepszona

File: death-valley-town/Content/translation/deathvalley/polish.json [hunk 2]
```diff
@@ -17,28 +17,28 @@
...
+    "building.death-valley-town.deathvalley.dwellingUpgrade4.name": "Ul. Nawiedzona Świątynia",
     "building.death-valley-town.deathvalley.dwellingUpgrade5.description": "Ulepszona siedziba Nocy pozwala rekrutować Nocne czarownice.",
     "building.death-valley-town.deathvalley.dwellingUpgrade5.name": "Ul. Siedziba Nocy",
     "building.death-valley-town.deathvalley.dwellingUpgrade6.description": "Ulepszona sala Ciemności pozwala rekrutować Ponurzych żniwiarzy.",
     "building.death-valley-town.deathvalley.dwellingUpgrade6.name": "Ul. Sala Ciemności",
-    "building.death-valley-town.deathvalley.dwellingUpgrade7.description": "Ulepszona świątynia Śmierci pozwala rekrutować Duchowe Golemy.",
-    "building.death-valley-town.deathvalley.dwellingUpgrade7.name": "Ul. Świątynia Śmierci",
+    "building.death-valley-town.deathvalley.dwellingUpgrade7.description": "Ulepszona Kanion Śmierci pozwala rekrutować Duchowe Golemy.",
```
Comment (ToRRent1812) [inline death-valley-town/Content/translation/deathvalley/polish.json:40]: Ulepszona -> Ulepszony

Comment (Fabrice-TIERCELIN) [reply death-valley-town/Content/translation/deathvalley/polish.json:40]: Thank you! I will fix it in a further PR 🙂

**Pull request merged**